core/src/main/web/app/helpers/globals.js

```diff
@@ -50,7 +50,22 @@
       THEMES: {
         DEFAULT: 'default',
         AMBIANCE: 'ambiance'
-      }
+      },
+      //see http://stackoverflow.com/questions/9847580/how-to-detect-safari-chrome-ie-firefox-and-opera-browser
```

Comment:
should go in helper.js, global.js is just for constants not for functions.

also, please only define firefox, safari, and chrome as we support only them.